### PR TITLE
Add ability to add an option to a metadata template multi select field

### PR DESCRIPTION
**Pull request by sujaygarlanka**

### CODE REVIEW

**Review by swfree — APPROVED** [review]

File: src/commands/metadata-templates/update.js [hunk 1]
```diff
@@ -301,6 +311,10 @@ MetadataTemplatesUpdateCommand.flags = {
 		description: 'Add a multi-select field with the provided display name',
 		multiple: true,
 	}),
+	'add-multi-select-option': flags.string({
+		description: 'Add an option to a specified multiselect field; must be followed by one or more --option flags',
```
Comment (PJSimon) [inline src/commands/metadata-templates/update.js:315]:
```suggestion
		description: 'Add an option to a specified multiselect field; must be followed by one or more --multi-select-option flags',
```

Comment (PJSimon) [reply src/commands/metadata-templates/update.js:315]: I think `option` might be too generic. Is this OK?

File: test/commands/metadata-templates.test.js [hunk 2]
```diff
@@ -488,6 +495,8 @@ describe('Metadata Templates', () => {
 				'--option=bar',
 				'--option=baz',
 				'--no-hidden',
+				'--add-multi-select-option=foo',
+				'--option=bas',
```
Comment (PJSimon) [inline test/commands/metadata-templates.test.js:499]:
```suggestion
				'--multi-select-option=bas',
```

File: src/commands/metadata-templates/update.js (hunk 1 above)
Comment (PJSimon) [reply src/commands/metadata-templates/update.js:315]: How are these `multi-select-option` flags tied to the usage of add-multi-select-option?  How does it work when there are multiple `multi-select-option` flags?

**Pull request merged**